### PR TITLE
Update homepage of 14 Casks

### PR DESCRIPTION
**Pull request by miccal**
Update homepage of 15 Casks.

Ping @victorpopkov 

This is from your updated list. After removing the ones that were simply local redirects and the ones you did in your PR [here](https://github.com/caskroom/homebrew-cask/pull/27086), there were 14 remaining.

I just swapped the two homepage in the `wercker` Casks.

The Cask `yasu` was a bit harder. There are versions for older MacOS's on their new website, but the app name (`Yasu.app` compared to `Yasu for Mac.app`) and download url's (see [here](https://yasuformac.com/download/)) are different, so I removed them.

### CODE REVIEW

File: Casks/yasu.rb [hunk 1]
```diff
@@ -1,17 +1,20 @@
 cask 'yasu' do
-  if MacOS.version <= :snow_leopard
-    version '2.8.2'
-    sha256 '427672a45b8315c2f38d968ea5e0c35c21b91091a1fe0e750fcd2b0078644336'
+  if MacOS.version <= '10.11'
```
Comment (victorpopkov) [inline Casks/yasu.rb]: It's better to change `'10.11'` to `:el_capitan` instead for consistency reasons.

File: Casks/yasu.rb [hunk 2]
```diff
@@ -1,17 +1,20 @@
...
+    version '504'
+    sha256 'f1bd1e48a2ff0e9839f4b21c651e89a4e18d8aa85a9b9fa8642f333ef5b0053b'
+    url "https://yasuformac.com/appcasts/10.11/yasuformac_#{version}.zip"
```
Comment (victorpopkov) [inline Casks/yasu.rb]: There is also an appcasts available for both:
 - https://yasuformac.com/appcasts/10.11/yasu.xml `1349d309621d03caef39dc1b2edadc0638926dd8c4899eef9073b19c7f59666b`
 - https://yasuformac.com/appcasts/10.12/yasu.xml
`5590958819d56aebfda1294628cb453117247a63355d7607e1e26de7264aea19`

File: Casks/yasu.rb [hunk 3]
```diff
@@ -1,17 +1,20 @@
 cask 'yasu' do
-  if MacOS.version <= :snow_leopard
-    version '2.8.2'
-    sha256 '427672a45b8315c2f38d968ea5e0c35c21b91091a1fe0e750fcd2b0078644336'
+  if MacOS.version <= '10.11'
+    version '504'
```
Comment (victorpopkov) [inline Casks/yasu.rb]: I would also include the version here alongside with the build and change the download URL accordingly.

- `3.0.4,504`
`https://yasuformac.com/appcasts/10.11/yasuformac_#{version.after_comma}.zip`
- `4.0.2,602`
`https://yasuformac.com/appcasts/10.12/yasuformac_#{version.after_comma}.zip`

File: Casks/yasu.rb [hunk 4]
```diff
@@ -1,17 +1,20 @@
 cask 'yasu' do
```
Comment (victorpopkov) [inline Casks/yasu.rb]: Since here are a lot of changes, I would recommend opening a separate PR for this cask.

**Review by victorpopkov — CHANGES_REQUESTED** [review]

**Comment by miccal** [comment]
Removed `yasu` and created a separate PR [here](https://github.com/caskroom/homebrew-cask/pull/27096).

**Review by victorpopkov — APPROVED** [review]

**Pull request merged**